docs/api-ref.md

```diff
@@ -4700,21 +4700,221 @@ See [ViewItem class](#viewitem-class)
+
+---
+
+ # create a webhook item
+ with server.auth.sign_in(tableau_auth):
+    new_webhook = TSC.WebhookItem()
+    new_webhook.name ='testWebhook'
+    new_webhook.event ="workbook-refresh-failed" # alternately, you can also use new_webhook.source="webhook-source-event-workbook-refresh-failed" 
+    new_webhook.url ="https://webhook.site/6e4c957d-dd40-422c-8fc6-7151afe7fc0b"    
+    # create the webhook
+    new_webhook = server.webhooks.create(new_webhook)
+    print("Webhook created. ID: {}".format(new_webhook.id))
+
+```
+<br>
+<br>
+
+### webhook.delete
+```py
+webhooks.delete(webhook_id)
+```
+
+Deletes a webhook by ID.
+
+To specify the site, create a `TableauAuth` instance using the content URL for the site `(site_id)` and sign in to the site. For more information on how to specify a site, see the [TableauAuth class](#tableauauth-class).
+
+REST API:  [Delete Webhook](https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref_notifications.htm#delete_webhook)
+
+**Parameters**
+
+Name|Description
+:---|:---
+`webhook_id`| The ID of the webhook to delete.
+
+**Exceptions**
+
+Error|Description
+:---|:---
+`Webhook ID undefined`| Raises an exception if a `webhook_id` is not provided. 
+
+**Example**
+```py
+# import tableauserverclient as TSC
+# server = TSC.Server('https://SERVER')
+# sign in . For authentication examples, see https://github.com/tableau/server-client-python/blob/master/samples/login.py
+---
+# Delete the webhook
+with server.auth.sign_in(tableau_auth):
+  server.webhooks.delete('7d60d364-b9f5-4a9c-8aa5-4bdaa38c5dd3')
+```
+<br>
+<br>
+
+#### webhook.get()
+```py
+webhooks.get()
+```
+Returns a list of all the webhooks for a site.  
+To specify the site, create a `TableauAuth` instance using the content URL for the site `(site_id)`, and sign in to that site. For more information on how to specify a site, see the [TableauAuth class](#tableauauth-class).
+
+REST API: [List Webhooks](https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref_notifications.htm#list_webhooks_for_site)
+
+
+**Parameters**
+
+Name |Description
+:---|:---
+`req_option`| (Optional) You can pass themthod oa request object that contains additional parameters to filter the request.  For example, you could specify the name of the webhook or the name of the owner. For more information, see [Filter and Sort](filter-sort).
+
+**Returns**
+
+Returns a list of all `ProjectItem` objects and a `PagainationItem`. Use  these values to iterate through the results. 
+
+**Example**
+
+```py
+# import tableauserverclient as TSC
+# server = TSC.Server('https://SERVER')
+# sign in . For authentication examples, see https://github.com/tableau/server-client-python/blob/master/samples/login.py
+---
+# get a list of all the webhooks on a site
+with server.auth.sign_in(tableau_auth):
+   all_webhooks, pagination_item = server.webhooks.get()
+print("\nThere are {} webhooks on site: ".format(pagination_item.total_available))
```

Comment:
add one blank line before here

Reply:
@bcantoni  - just to make sure I understand, are you asking to add a blank line between the two print statements?

Reply:
right - basically adding a blank line after the indented code block, so for this snippet it would look like:

```
# get a list of all the webhooks on a site
with server.auth.sign_in(tableau_auth):
   all_webhooks, pagination_item = server.webhooks.get()

print("\nThere are {} webhooks on site: ".format(pagination_item.total_available))
```